README.md

```diff
@@ -4,26 +4,55 @@ Applications on Autopilot
 *A demonstration of the autopilot pattern for self-operating microservices, designed as a workshop or tutorial.*
 
 
-Microservices and Docker go together like chocolate and peanut butter: microservice architectures provide organizations a tool to manage complexity of the development process, and application containers provide a new means to manage the dependencies and deployment of those microservices. But deploying and connecting those services together is still a challenge because it forces developers to design for operationalization.
+Microservices and [Docker](https://www.docker.com) go together like chocolate
```

Comment:
On our autopilotpattern repos we don't wrap the markdown at 80 cols. Call it a Product-team quirk.